setup.py

```diff
@@ -97,6 +97,7 @@
         # extracting injected parameters from notebooks
         "parso",
         'importlib-metadata;python_version<"3.8"',
+        "altair"
```

Comment:
let's make this an optional dependency https://ploomber-core.readthedocs.io/en/latest/dependencies.html